conda/gateways/repodata/jlap/fetch.py

```diff
@@ -0,0 +1,402 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+# Lappin' up the jlap
+from __future__ import annotations
+import io
+
+import json
+import logging
+import pathlib
+import pprint
+import re
+import time
+import zstandard
+from contextlib import contextmanager
+from hashlib import blake2b
+from typing import Iterator
+
+import jsonpatch
+from requests import HTTPError
+
+from conda.base.context import context
+from conda.gateways.connection import Response, Session
+from conda.gateways.repodata import RepodataState
+
+from .core import JLAP
+
+log = logging.getLogger(__name__)
+
+
+DIGEST_SIZE = 32  # 160 bits a minimum 'for security' length?
```

Comment:
This is also defined in `conda.gateways.repodata.jlap.core`, would it help to move all of the jlap constants into a `conda.gateways.repodata.jlap.constants` submodule?

Reply:
This digest size is used for a different purpose (digest of the reconstituted repodata.json file) and the jlap `core` keyed hash digest size is used for the integrity of the individual lines in the jlap file. They don't necessarily have to be the same size, even though they are in this application.

Reply:
If they're used for different things and mean different things they need to be called different things. Including a docstring would also help.

Reply:
On the one hand I see what you mean, on the other hand the context is that they are in different files. Can rename the constant.